frameworks/hdfs/universe/config.json

```diff
@@ -1829,6 +1829,10 @@
         "hadoop_util_hash_type": {
           "type": "string",
           "description": "The default implementation of Hash. Currently this can take one of the two values: 'murmur' to select MurmurHash and 'jenkins' to select JenkinsHash.",
+          "enum": [
```

Comment:
Please also add enums for 
- `disk_type`
- [logging_level](https://github.com/mesosphere/dcos-commons/blob/45625cee242f42988833d987087babe5cdd23481/frameworks/hdfs/universe/config.json#L299)
- [client_block_write_replace-datanode-on-failure_policy](https://github.com/mesosphere/dcos-commons/blob/45625cee242f42988833d987087babe5cdd23481/frameworks/hdfs/universe/config.json#L1200)

Reply:
done